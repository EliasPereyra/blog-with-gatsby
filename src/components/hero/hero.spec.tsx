/**
 * @jest-environment jsdom
 */

import * as React from "react";
import { render } from "@testing-library/react";
import Hero from ".";

describe("Hero", () => {
  it("Renders succesfully", () => {
    const hero = render(<Hero />);

    expect(hero).toMatchSnapshot();
  });
});
