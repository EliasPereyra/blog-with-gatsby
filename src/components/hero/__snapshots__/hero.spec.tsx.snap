// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Renders succesfully 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section>
        <h1
          style="max-width: 320px;"
        >
          Welcome to my blog
        </h1>
        <p
          style="margin-bottom: 48px;"
        >
          Know everything about software development
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <section>
      <h1
        style="max-width: 320px;"
      >
        Welcome to my blog
      </h1>
      <p
        style="margin-bottom: 48px;"
      >
        Know everything about software development
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
